<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SupplierForm</class>
 <widget class="QDialog" name="SupplierForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>629</width>
    <height>530</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:green;</string>
  </property>
  <widget class="QPushButton" name="save">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>380</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black;
background-color: lightblue</string>
   </property>
   <property name="text">
    <string>Сохранить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="previous">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>370</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black;
background-color: lightblue</string>
   </property>
   <property name="text">
    <string>Назад</string>
   </property>
  </widget>
  <widget class="QPushButton" name="next">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>370</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:black;
background-color: lightblue</string>
   </property>
   <property name="text">
    <string>Вперед</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>340</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black;
background-color: lightblue</string>
   </property>
   <property name="text">
    <string>Добавить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="delete_2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>420</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black;
background-color: lightblue</string>
   </property>
   <property name="text">
    <string>Удалить</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="supplier_id">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>170</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
color:black;
font-size: 16px</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="supplier_name">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>230</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
color:black;
font-size:16px</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>50</y>
     <width>231</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
font-size: 36px</string>
   </property>
   <property name="text">
    <string>Поставщики</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>170</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
font-size: 16px</string>
   </property>
   <property name="text">
    <string>ID Поставщика</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>230</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
font-size: 16px</string>
   </property>
   <property name="text">
    <string>Название</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
